lib/spi/gitops-contracts.ts

```diff
@@ -8,7 +8,7 @@ import { GitRepositoryReference, Values } from ".";
 export interface GitOpsApplication {
     name: string,
     repository?: GitRepositoryReference,
-    namespace: string
```

Comment:
please remind me why this change as opposed to default to `default` namespace? Is it because namespace is set in GitOps?

Reply:
@shapirov103 , I had to change this from `string` to `string | undefined`  to match the one defined in this type:
https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/lib/addons/helm-addon/kubectl-provider.ts#L16

I think this might be related to this comment here https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/lib/addons/container-insights/index.ts#L15